packages/core/src/components/ui/VirtualTable/_VirtualTable.scss

```diff
@@ -0,0 +1,58 @@
+@import "../../../common/styles/colors";
+
+.ReactVirtualized__Table__headerRow {
+  border-bottom: none;
+  padding-right: 0 !important;
+  height: 48px;
+}
+
+.ReactVirtualized__Table__headerColumn {
+  font-weight: normal;
+
+  > * {
+    font-size: 11px !important;
+  }
+}
+
+.ReactVirtualized__Table__headerTruncatedText {
+  margin-right: 4px;
+  font-weight: bold;
+  font-size: 13px;
+  text-transform: none;
+}
+
+.ReactVirtualized__Table__row {
+  color: $white;
```

Comment:
for ufx-ui, default theme is light/white, so i think text in white wont be visible when we use this in ufx-components,  can you change to  $ufx-text-color here and then customise/overwrite it for in hf-ui scss if required